studygroups/views/api.py

```diff
@@ -355,14 +407,21 @@ def get(self, request):
         query_schema = {
             "offset": schema.integer(),
             "limit": schema.integer(),
-            "order": lambda v: (v, None) if v in ['title', 'usage', 'overall_rating', 'created_at', None] else (None, "must be 'title', 'usage', 'created_at', or 'overall_rating'")
+            "order": lambda v: (v, None) if v in ['title', 'usage', 'overall_rating', 'created_at', None] else (None, "must be 'title', 'usage', 'created_at', or 'overall_rating'"),
+            "user": schema.text(),
+            "include_unlisted": schema.text(),
         }
         data = schema.django_get_to_dict(request.GET)
         clean_data, errors = schema.validate(query_schema, data)
         if errors != {}:
             return json_response(request, {"status": "error", "errors": errors})
 
-        courses = Course.objects.active().filter(unlisted=False).annotate(
+        courses = Course.objects.active()
+
+        if request.GET.get('include_unlisted', "false") == "false":
```

Comment:
This should only be available to signed in users. And only for their own courses , iow when the user is given and the user given is the user logged in.